### PR TITLE
pytest ignore two failing tests

### PR DESCRIPTION
**Pull request by cclauss**
Make pytest mandatory except for the two failing tests mentioned in #2325

### CODE REVIEW

File: .github/workflows/test_python.yml [hunk 1]
```diff
@@ -15,5 +15,9 @@ jobs:
       - run: pip install .
       - run: mkdir --parents --verbose .mypy_cache
       - run: mypy --ignore-missing-imports --install-types --non-interactive . || true
+      - run: pytest --ignore=tests/test_utils/test_mjpeg.py
+                    --ignore=tests/test_utils/test_rtmp.py .
+      - run: pytest --fixtures tests/test_utils/test_mjpeg.py || true
+      - run: pytest --fixtures tests/test_utils/test_rtmp.py  || true
       - run: pytest . || pytest --doctest-modules . || true
```
Comment (MichaIng) [inline .github/workflows/test_python.yml:22]: This line is then obsolete, isn't it?

Comment (cclauss) [reply .github/workflows/test_python.yml:22]: I want to be able to monitor if the tests get fixed.

Comment (MichaIng) [reply .github/workflows/test_python.yml:22]: Isn't this seen by the two above commands already? I mean currently you run all expectedly succeeding tests, then the expectedly failing tests, then again all tests.

Comment (cclauss) [reply .github/workflows/test_python.yml:22]: There are tests in test_mjpeg.py and test_mjpeg.py that pass and others that fail.

https://docs.python.org/3/library/doctest.html are useful but we currently have none.

Comment (MichaIng) [reply .github/workflows/test_python.yml:22]: I should have looked at the tests output. Learned now that `pytest --fixtures` doesn't actually run the test. All good then 👍.

**Review by MichaIng — APPROVED** [review]

**Pull request merged**